sdk/program/src/lib.rs

```diff
@@ -444,9 +444,9 @@
 //!   - Instruction: [`solana_sdk::secp256k1_instruction`](https://docs.rs/solana-sdk/latest/solana_sdk/secp256k1_instruction/index.html)
 //!   - Invokable by programs? no
 //!
-//! - __secp256r1 Program__: Verifies an secp256r1 signature.
-//!   - ID: [`solana_program::secp256r1_program`]
-//!   - Instruction: [`solana-secp256r1`](https://docs.rs/solana-secp256r1)
+//! - __secp256r1 Program__: Verifies a secp256r1 signature.
+//!   - ID: [`solana_secp256r1_program::id`]
+//!   - Instruction: [`solana_secp256r1`](https://docs.rs/latest/solana-secp256r1/index.html)
```

Comment:
```suggestion
//! - __secp256r1 Program__: Verifies a secp256r1 signature.
//!   - ID: [`solana_secp256r1_program`]
//!   - Instruction: [`solana_secp256r1_program`](https://docs.rs/latest/solana-secp256r1/index.html)
```

Reply:
The `::id` is kind of pedantic, but I'm just looking at the patterns above it.

Reply:
I also realized my original comment from my first review on this was wrong, heh my bad. Should be `solana_secp256r1_program`.

Reply:
Fixing shortly 👍 I'm a bit confused, `ci/test-sanity.sh` passes locally and doesn't seem to catch the error in the docs.

Reply:
Fixed the above recommendations  and the  trailing whitespace from previous CI error in [docs fixes](https://github.com/anza-xyz/agave/pull/3152/commits/30dbde6a0880f4b566bd858a0c354b71a65a427e) ✅